GoogleChromeLabs/ps-analysis-tool
# Feature: Add filters to stories

## Description
This PR creates dumb components (TopBar, Chips, FiltersSidebar) from Table components so they can be reused across other panels. 
The stories panel has implemented these components with dummy data.
Also refactoring the code for the table to include these new dumb components has been done.


## Relevant Technical Choices


## Testing Instructions
- Open the PSAT panel and go to the Stories panel.
- A Search bar, Filter sidebar, and Chips bar should be present.
- The components should be working but no effect on data for now.
- Now move to the Cookies Panel, landing and tables both individually.
- Test the functionality of filters, it should be working the same as compared to the develop branch.


## Additional Information:



## Screenshot/Screencast



---

## Checklist






Fixes #
